services/asset/nodes/schemas.go

```diff
@@ -124,30 +136,72 @@ func makeNodeSchema(recursiveNodes map[string]*schema.Schema) map[string]*schema
 }
 
 var dataSourceFilterSchema = map[string]*schema.Schema{
-	"parent_node_ids": {
+	"created_by": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the user who created the Node",
+	},
+	"from_created_at": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the Node creation date, using ISO-8601 format. Properties with a creation date greater or equals than the filter value will be selected (if they are not excluded by other filters)",
```

Comment:
not a blocker, but could you add validations for the fields that are times? This way the description does not need to contain the ISO-8601 information

Reply:
I couldn't use the function IsValidTimeDateOrTimestamp with empty strings (I get an error that because empty strings aren't allowed... I don't have this behaviour with Schemas, so I guess it is linked to data source types) : 
```
data "leanspace_nodes" "all" {
  filters {
    parent_node_ids = []
    types           = ["ASSET"]
    is_root_node    = true
    created_by            = ""
    from_created_at       = ""
    last_modified_by      = ""
    to_created_at         = ""
    from_last_modified_at = ""
    to_last_modified_at   = ""
    ids                   = []
    kinds                 = []
    query                 = ""
    tags                  = []
    page                  = 0
    size                  = 10
    sort                  = ["name,asc"]
  }
}
```

To allow empty strings, I created the IsEmptyOrValidTimeDateTimestamp function. 

I could use IsValidTimeDateOrTimestamp if I set the fields to null :  
```
data "leanspace_nodes" "all" {
  filters {
    parent_node_ids = []
    types           = ["ASSET"]
    is_root_node    = true
    created_by            = ""
    from_created_at       = null
    last_modified_by      = ""
    to_created_at         = null
    from_last_modified_at = null
    to_last_modified_at   = null
    ids                   = []
    kinds                 = []
    query                 = ""
    tags                  = []
    page                  = 0
    size                  = 10
    sort                  = ["name,asc"]
  }
}
```
I prefer this solution but would it confuse the users to put null instead of an empty string ?

I have the same behaviour for UUID validation (so I created IsValidUUIDOrEmpty function).

Reply:
I guess it's ok to fail if the string is empty because it's not valid
Just omitting it or putting it to null should be the correct behaviour but I suppose it should be documented correctly

Reply:
Ok I will remove the function IsEmptyOrValidTimeDateTimestamp & IsValidUUIDOrEmpty to put  IsEmptyOrValidTimeDateTimestamp & IsUUID instead. I will add some explanation in the schema.go file to make things clear.